.github/workflows/fedora-copr-build.yml

```diff
@@ -129,6 +129,18 @@ jobs:
             --delete-after-days 32 \
             $chroot_opts "${{ env.project_today }}"
 
+      - name: "Enable swig:4.0 module in RHEL 8 build chroots (if any)"
+        shell: bash -e {0}
+        run: |
+          source scripts/functions.sh
+          [[ ! -z "${{ matrix.extra_script_file }}" ]] && source ${{ matrix.extra_script_file }}
+
+          for chroot in ${{ env.chroots }}; do
+            if [[ "$chroot" ==  rhel-8- ]]; then
```

Comment:
```suggestion
            if [[ "$chroot" == rhel-8-* ]]; then
```
I think?

Reply:
Totally. In fact, see my console for my worst tries today. I had this but forgot to write it down properly. Thank you for finding this.

![grafik](https://github.com/user-attachments/assets/9316c018-60fc-4e9b-82be-63b5158a6685)